nebula/ui/components/VPNTutorialCard.qml

```diff
@@ -50,39 +49,29 @@ VPNCard {
             }
         }
 
-
         ColumnLayout {
-            Layout.topMargin: 16
+            Layout.topMargin: VPNTheme.theme.vSpacing * 0.5
             Layout.rightMargin: 16
-            Layout.bottomMargin: 8
+            Layout.bottomMargin: VPNTheme.theme.vSpacing * 0.5
+            Layout.fillHeight: true
+            Layout.fillWidth: true
 
             spacing: 0
 
-            VPNBoldInterLabel {
-                id: cardTypeText
-
-                Layout.fillWidth: true
-
-                text: root.cardTypeText
-                elide: Text.ElideRight
-                verticalAlignment: Text.AlignVCenter
-                maximumLineCount: 1
-
-                Accessible.ignored: true
+            Item {
+                Layout.fillHeight: true
             }
 
-
             VPNBoldLabel {
                 id: titleText
 
-                Layout.topMargin: VPNTheme.theme.listSpacing
                 Layout.fillWidth: true
 
                 elide: Text.ElideRight
```

Comment:
Do we need to set a max-width in order for the elide to work?

Reply:
`Layout.fillWidth` does the trick

<img width="348" alt="Screen Shot 2022-06-10 at 4 43 41 PM" src="https://user-images.githubusercontent.com/15353801/173147566-654a8993-b22b-434b-a517-0644602b9097.png">